src/devices/src/virtio/block/disk/mod.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 The ChromiumOS Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
Is this file a new one?, if that is the case, why `Copyright 2019 The ChromiumOS Authors`?

Reply:
You're right. If it comes from crosvm, we need to keep the copyright notice.

Reply:
@slp As of my latest push, `disk/mod.rs` just contains `QCOW_MAGIC` and `ImageType`. Does that still justify the copyright notice, or can I remove it?